### DISCUSSION TITLE
v3.9.1

### DISCUSSION THREAD

**Discussion by mattieserver** [mattieserver/netbox-topology-views#481] [Announcements]
## What's Changed
* Closes #329: Draw rectangles around Sites/Locations/Racks by @dreng in https://github.com/netbox-community/netbox-topology-views/pull/462
* Fixes #463: Pull 462 causes export xml to fail by @dreng in https://github.com/netbox-community/netbox-topology-views/pull/466
* Fixes #471: Group Locations frame and font color hardly readable with light theme by @dreng in https://github.com/netbox-community/netbox-topology-views/pull/472
* Fixes #473: Show Power Feeds raises AttributeError by @dreng in https://github.com/netbox-community/netbox-topology-views/pull/474
* Fixes #236: Saved filter doesn't reload show cables by @dreng in https://github.com/netbox-community/netbox-topology-views/pull/475
* Fix #303: Server Error after adding an Object Count Widget by @dreng in https://github.com/netbox-community/netbox-topology-views/pull/476
* prepare 3.9.1 by @mattieserver in https://github.com/netbox-community/netbox-topology-views/pull/480

**Full Changelog**: https://github.com/netbox-community/netbox-topology-views/compare/v3.9.0...v3.9.1

<hr /><em>This discussion was created from the release <a href='https://github.com/netbox-community/netbox-topology-views/releases/tag/v3.9.1'>v3.9.1</a>.</em>

**Comment by Cooldog1099** [comment]
Is this compatible with Netbox v4.0.1?

**Reply by mattieserver** [reply]
See #477 for when support for 4.0 will be added

**Comment by dreng** [comment]
> Is this compatible with Netbox v4.0.1?

Nope
